quickwit-core/src/index.rs

```diff
@@ -353,17 +353,15 @@ pub async fn remove_indexing_directory(data_dir_path: &Path, index_id: String) -
 
 /// Resolve storage endpoints to validate.
 pub async fn validate_storage_uri(
-    metastore_uri_resolver: &MetastoreUriResolver,
```

Comment:
Here we validate a storage URI with a `metastore_uri_resolver`.... am I missing something?